### ISSUE TITLE
Fix SEO for Angular

### ISSUE THREAD

**Issue by BorntraegerMarc** [angular/angular#18200]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Feature request
[X] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
</code></pre>

## Current behavior

Always when I search on google something for "Angular" (which is the official name for angular 2+) I end up on AngularJS (AngularJS 1) results. I always need to add the suffix "2".

So I end up googling: "Angular 2 service", which also brings me results for Angular 4.

## Expected behavior

When searching in search engines after "Angular" it should return "Angular 2+" results

## Minimal reproduction of the problem with instructions

Example: https://www.google.ch/search?q=angular+service&oq=angular+service&aqs=chrome..69i57j69i60j35i39l2j0l2.3718j0j7&sourceid=chrome&ie=UTF-8

The search results only contain AngularJS.

## What is the motivation / use case for changing the behavior?

You want to be recognized as "It's just angular" which will never happen if all people have to search for "Angular 2" on search engines.

## Environment

<pre><code>
Angular version: X.Y.Z

Browser:
- [ ] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: XX  
- Platform:  

Others:

</code></pre>

**Comment by Toxicable** [comment]
Duplicate #14802
TLDR: use `-angularjs` to the end of your searches when looking for content for Angular
